docs/index.md

```diff
@@ -0,0 +1,45 @@
+## Introduction
+
+Welcome to the `vscode-debug-adapter-apache-camel` project created by Red Hat! Here you'll find a description of how to use VS Code extension providing Debug Adapter for Apache Camel.
+
+## Description
+
+This extension adds <a href="https://camel.apache.org/manual/debugger.html">Camel Debugger</a> power by attaching to a running Camel route written in Java, Yaml or XML DSL directly in your Visual Studio Code editor. It is working as client using the [Microsoft Debug Adapter Protocol](https://microsoft.github.io/debug-adapter-protocol/) which communicates with [Camel Debug Server](https://github.com/camel-tooling/camel-debug-adapter) providing all functionalities.
+
+## How to install
+
+1. You can download this **vscode-debug-adapter-apache-camel** extension from the [VS Code Extension Marketplace](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-debug-adapter-apache-camel) and the [Open VSX Registry](https://open-vsx.org/extension/redhat/vscode-debug-adapter-apache-camel).
+2. Debug Adapter for Apache Camel can be also installed directly in the [Microsoft VS Code](https://code.visualstudio.com/).
+
+    **Steps**
+    - Open your VS Code.
+    - In VS Code, select **View > Extensions**.
+    - In the search bar, type `Camel`
+    - Select the **Debug Adapter for Apache Camel** option and then click `Install`.
+
+## Features
+
+- Add/Remove breakpoints
+- Support a single Camel context
+- `Camel Main` mode for XML only
+  - Support use of Camel debugger
+  - Support local use of Camel debugger
```

Comment:
1) "Camel Main mode" before inside list, so
Camel Main mode ... Support -> Camel Main mode ... Supports

2) local use of Camel debugger -> local use of **the** Camel debugger